src/daemon/control.rs

```diff
@@ -493,7 +493,19 @@ pub fn handle_rpc_messages(
                     ..
                 } = db_tip(&db_path)?;
 
-                response_tx.send((network.to_string(), blockheight, progress))?;
+                let number_of_funded_vaults =
+                    listvaults_from_db(&revaultd.read().unwrap(), None, None)?
+                        .iter()
+                        .filter(|l| l.status == VaultStatus::Funded)
```

Comment:
This would discard all vaults that are not **only** funded (ie active, unvaulting, w/e). I suggest that we go with `l.status != VaultStatus::Spent` and just rename the field `vaults` in the `getinfo` result.

Reply:
Ah, ok, thought your comment meant to count only the funded :) I agree, counting the non spent makes more sense